src/lib/containers/oauth/OAuthManagement.tsx

```diff
@@ -33,7 +49,17 @@ export const OAuthManagement: React.FunctionComponent = () => {
                   {item.verified ? 'Yes' : 'SUBMIT_VERIFICATION_PLACE_HOLDER'}
                 </td>
                 <td>GENERATE_PLACEHOLDER</td>
-                <td>ACTIONS_PLACEHOLDER</td>
+                <td>
+                  <button
+                    onClick={() => {
+                      setIsShowingCreateClientModal(true)
+                      setSelectedClient(item)
+                      setIsEdit(true)
```

Comment:
I think these will all get batched into one update, but updating the data before showing the modal is a little safer 
```suggestion
                      setSelectedClient(item)
                      setIsEdit(true)
                      setIsShowingCreateClientModal(true)
```